ionoscloud/resource_server.go

```diff
@@ -407,6 +407,22 @@ func resourceServer() *schema.Resource {
 				Optional: true,
 				Elem:     labelResource,
 			},
+			// When deleting the server, we need to delete the volume that was defined INSIDE the
+			// server resource. The only way to know which volume was defined INSIDE the server
+			// resource is to save the volume ID in this list.
+			// NOTE: In the current version, v6.3.6, it is required to define one volume (and only
+			// one) inside the server resource, but in the future we consider the possibility of
+			// adding more volumes within the server resource, in which case the current code should
+			// be revised as changes should also be made for the update function, as the list of
+			// inline_volume_ids can be modified.
+			"inline_volume_ids": {
+				Type:        schema.TypeList,
+				Description: "A list that contains volumes defined inside the server resource.",
```

Comment:
`a list that contains volume ids`

Reply:
@cristiGuranIonos done in ab13150538338f9b0acb582129addfa57e182153